apps/avifenc.c

```diff
@@ -2121,9 +2125,9 @@ int main(int argc, char * argv[])
         avifInputFileSettings * fileSettings = &file->settings;
 
         // Check tiling parameters.
-        // Auto tiling (autoTiling) and manual tiling (tileRowsLog2, tileColsLog2) are mutually exclusive, which means:
-        // - At each input, only one of the two shall be set.
-        // - At some input, specify one disables the other.
+        // Automatic tiling (autoTiling) and manual tiling (tileRowsLog2, tileColsLog2) are mutually exclusive, which means:
+        // - At each input, only one of the two may be set.
```

Comment:
The original word "shall" seems to imply that either auto tiling or manual tiling must be set at each input. I changed "shall" to "may" to suggest that it is optional to set one of the two tiling modes.